release/restore.sh

```
@@ -0,0 +1,113 @@
+            else
+            if [ -f /usr/share/tomcat7/bin/version.sh ] ; then
+                TOMCAT=tomcat7
+            fi
+        fi
+    fi
+fi
+
+TMP=/tmp/${TOMCAT}-${TOMCAT}-tmp
+data_path=/usr/share/oscar-emr
+PROGRAM=oscar
+LOG_FILE=${data_path}/${PROGRAM}.log
```

Shellcheck (0.11.0)

[warning] LOG_FILE appears unused. Verify use (or export if used externally).

(SC2034)